ArduPlane/mode_guided.cpp

```diff
@@ -119,6 +119,37 @@ bool ModeGuided::handle_guided_request(Location target_loc)
     return true;
 }
 
+#if AP_PLANE_OFFBOARD_GUIDED_SLEW_ENABLED
+MAV_RESULT ModeGuided::handle_change_airspeed(const float airspeed, const float acceleration) {
+    // reject airspeeds that are outside of the tuning envelope
+    if (airspeed > plane.aparm.airspeed_max || airspeed < plane.aparm.airspeed_min) {
+        return MAV_RESULT_DENIED;
+    }
+
+    // no need to process any new packet/s with the
+    // same airspeed any further, if we are already doing it.
+    float new_target_airspeed_cm = airspeed * 100;
+    if ( is_equal(new_target_airspeed_cm,plane.guided_state.target_airspeed_cm)) { 
+        return MAV_RESULT_ACCEPTED;
```

Comment:
exits early without setting acceleration and timestamp.  I think you might have mentioned this

Reply:
Yea, do you want a functional change in this refactor too? I'd prefer to solve that in a separate PR with a test that verifies it's working.